common/next/next.config.js

```diff
@@ -7,102 +7,103 @@ const withMDX = require('@next/mdx')({
-                  destination: `${identityHost}/logout`,
-                  permanent: true,
-                },
-              ];
-            }
-            return [];
-          },
-          webpack: (config, { isServer, webpack }) => {
-            const bundleAnalysisFile = `../../.dist/${
-              isServer ? 'server' : 'client'
-            }.${buildHash}`;
+        },
+        async rewrites() {
+          if (phase === PHASE_DEVELOPMENT_SERVER) {
+            return [
+              {
+                source: '/api/users/me',
+                destination: `${identityHost}/api/users/me`,
+              },
+              {
+                source: '/api/users/:user_id/item-requests',
+                destination: `${identityHost}/api/users/:user_id/item-requests`,
+              },
+            ];
+          }
+          return [];
+        },
+        async redirects() {
+          if (phase === PHASE_DEVELOPMENT_SERVER) {
+            return [
+              {
+                source: '/account',
+                destination: `${identityHost}`,
+                permanent: true,
+              },
+              {
+                source: '/account/logout',
+                destination: `${identityHost}/logout`,
+                permanent: true,
+              },
+            ];
+          }
+          return [];
+        },
+        webpack: (config, { isServer, webpack }) => {
+          config.plugins.push(
+            new webpack.NormalModuleReplacementPlugin(
+              /moment-timezone\/data\/packed\/latest\.json/,
+              path.join(__dirname, 'timezones.json')
+            )
+          );
+
+          if (shouldAnalyzeBundle) {
+            // This path is relative to the .next directory
+            const bundleEnvironment = isServer ? 'server' : 'client';
+            const bundleAnalysisFile = `../.dist/${options.applicationName}.${bundleEnvironment}.${buildHash}`;
 
             config.plugins.push(
-              new webpack.NormalModuleReplacementPlugin(
-                /moment-timezone\/data\/packed\/latest\.json/,
-                path.join(__dirname, 'timezones.json')
-              )
+              new BundleAnalyzerPlugin({
+                analyzerMode: 'static',
+                generateStatsFile: true,
+                openAnalyzer: false,
+                statsFilename: `${bundleAnalysisFile}.json`,
+                reportFilename: `${bundleAnalysisFile}.html`,
+              })
             );
+          }
 
-            if (shouldAnalyzeBundle) {
-              config.plugins.push(
-                new BundleAnalyzerPlugin({
-                  analyzerMode: 'static',
-                  generateStatsFile: true,
-                  openAnalyzer: false,
-                  statsFilename: `${bundleAnalysisFile}.json`,
-                  reportFilename: `${bundleAnalysisFile}.html`,
-                })
-              );
-            }
+          config.module.rules.push({
+            test: /\.css$/,
```

Comment:
do we need these still?

Reply:
Good question, I just copied them over in https://github.com/wellcomecollection/wellcomecollection.org/pull/7032. I'll try removing them in a separate PR as this contains an unnerving amount of build changes already